scipy/stats/_discrete_distns.py

```diff
@@ -514,6 +514,121 @@ def _logcdf(self, k, M, n, N):
 hypergeom = hypergeom_gen(name='hypergeom')
 
 
+class nhypergeom_gen(rv_discrete):
+    r"""A negative hypergeometric discrete random variable.
+
+    The negative hypergeometric distribution models the probability
+    of :math:`k` successes in a sample of exactly :math:`r` failures
+    taken without replacement where :math:`N` are the total number
+    of elements and :math:`K` are defined to be successes.
+
+    It is equivalent to observing :math:`k` successes in :math:`k+r-1`
+    samples with :math:`k+r`'th sample being a failure. The former
+    can be modelled as a hypergeometric distribution. Refer
+    `hypergeom` distribution for more information.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The symbols used to denote the shape parameters (`N`, `K`, and `r`) are not
+    universally accepted. See the Examples for a clarification of the
+    definitions used here.
+
+    The probability mass function is defined as,
+
+    .. math:: f(k; N, K, r) = \frac{{{k+r-1}\choose{k}}{{N-r-k}\choose{K-k}}}
+                                   {{N \choose K}}
+
+    for :math:`k \in 0, 1, 2, ..., K`, where the binomial
+    coefficients are defined as,
+
+    .. math:: \binom{n}{k} \equiv \frac{n!}{k! (n - k)!}.
+
+    The cumulative distribution, survivor function, hazard function,
```

Comment:
You could mention this in the tutorial. I suggest to remove it here as not relevant for the implementation

Reply:
Yeah! I will write a tutorial and mention there